api-specificatie/zrc/openapi.yaml

```diff
@@ -72,83 +75,95 @@ paths:
-        omschrijving:
-          description: Omschrijving of toelichting op de zaak
-          type: string
-        initiator:
-          $ref: '#/components/schemas/Subject'
+        url:
+          title: Url
+          type: string
+          format: uri
+          readOnly: true
+        zaakidentificatie:
+          title: Zaakidentificatie
+          description: >-
+            De unieke identificatie van de ZAAK binnen de organisatie die
+            verantwoordelijk is voor de behandeling van de ZAAK.
+          type: string
+          maxLength: 40
+        zaaktype:
+          title: Zaaktype
+          description: URL naar het zaaktype in de CATALOGUS waar deze voorkomt
+          type: string
+          format: uri
+          maxLength: 200
+          minLength: 1
         status:
-          description: URL naar de status van de zaak
+          title: Status
+          description: 'Indien geen status bekend is, dan is de waarde ''null'''
           type: string
+          format: uri
+          readOnly: true
+        omschrijving:
+          title: Omschrijving
+          description: Een korte omschrijving van de zaak.
+          type: string
+          maxLength: 80
+        # TODO: in het RGBZ 2.0 model loopt dit via de rollen/betrokkenen
+        # initiator:
+        #   title: Initiator
+        #   description: URL naar de initiator van de ZAAK.
+        #   type: string
+        #   format: url
         registratiedatum:
-          description: Tijdstip waarop de zaak is geregistreerd
-          type: string
-          format: 'date-time'
-        kanaal:
-          description: Kanaal via welke de zaak is gemeld
-          type: string
-        locatie:
-          description: Coordinate van de locatie waarop de zaak betrekking heeft in Well Known Text (WKT) formaat
+          title: Registratiedatum
+          description: De datum waarop de zaakbehandelende organisatie de ZAAK heeft geregistreerd
+          type: string
+          format: 'date'
+        # TODO: volgens RGBZ 2.0 komt kanaal uit KLANTCONTACT
+        # kanaal:
+        #   description: Kanaal via welke de zaak is gemeld
+        #   type: string
+        zaakgeometrie:  # NOTE: komt ook voor bij OBJECT
+          title: Zaakgeometrie
+          description: Punt, lijn of multi-vlak geometrie-informatie, in WKT formaat.
           type: string
         objecten:
+          title: OBJECTen
           description: Lijst met URL's naar aan de zaak gerelateerde objecten
           type: array
           items:
             type: string
-        zaakgegevens:
-          description: Domein specifieke gegevens beheorende bij de zaak
-          type: array
-          items:
-            $ref: '#/components/schemas/Zaakgegeven'
-      required:
-      - type
-      - omschrijving
-    ZaakId:
-      description: zaakidentificatie
-      properties:
-        zaakidentificatie: 
-          type: string
-          description: zaakidentificatie
-    Subject:
```

Comment:
Vraag me af of dit uiteindelijk wel in het ZRC moet. Hier hebben we het BRP toch voor... 😞

We gaan wel meer van dit soort registraties tegen komen die eigenlijk niet bij (puur) zaken horen. Denk dat het een Slack vraag is wat we hiermee moeten doen. Kan wellicht in Overige Registraties Component (ORC).

Reply:
Dat was ook mijn gedachte - ik denk dat een `Betrokkene` straks 'gewoon' een url naar een BRP resource wordt, en die komt straks (ter support) in ORC.

Alleen moeten we even kijken wat dat betekent qua concreet gebruik, want als er geen BRP is, dan rekt dat de scope op tot ook een OAS 3.0 specificatie van BRP... :worried:

Reply:
Ja, ik denk ook dat met URL's gaat verwijzen naar andere resources zoals personen, bedrijven, BAG-objecten.

Voor domeinspecifieke gegevens moeten we volgens mij alvast een oplossingsrichting kiezen, omdat we er direct mee te maken krijgen door US water.

Hugo